content/efcore/getting-started/manipulating-entities.md

```diff
@@ -132,7 +132,4 @@ using (var context = new LibraryContext())
  
 ## Deleting Related Data 
  
-In the previous example, we were able to successfully delete a book, but what happened to that book's checkout records? The short answer is it depends. The way that EF Core handles deletion of an entity with related data depends on several factors including whether the related data is tracked, how EF Core is configured, and how the database is configured. To learn more about the available behaviors and how to configure a certain behavior, check out the [Cascade Delete]() lesson.  
- 
-> **Note** {.note}  
-> The Cascade Delete lesson has not been added yet. For now, please refer to the [Cascade Delete](https://docs.microsoft.com/en-us/ef/core/saving/cascade-delete) page in the docs.
+In the previous example, we were able to successfully delete a book, but what happened to that book's checkout records? The short answer is it depends. The way that EF Core handles deletion of an entity with related data depends on several factors including whether the related data is tracked, how EF Core is configured, and how the database is configured. We learn more about the available behaviors and how to configure a certain behavior in the next lesson.
```

Comment:
We *will* learn about the available deletion behaviors and how to configure *them* in the next lesson.